summary_workflows/quantification/match_with_gt.py

```diff
@@ -0,0 +1,133 @@
+import subprocess
+from io import StringIO
+import pandas as pd
+import numpy as np
+from shutil import which
+import argparse
+
+parser = argparse.ArgumentParser()
+
+parser.add_argument('a', help='The BED file containing predictions. MUST be BED6 format.')
+parser.add_argument('b', help='The ground truth bed file. First 6 columns must be standard BED6, but can have additional columns appended.')
+parser.add_argument('window', help='Number of bases to append to each side of the predicted site.', type=int)
+#parser.add_argument('-o', help='output file directory') # not yet implemented!!
```

Comment:
Instead of output directory, you could also give the option to save output to specified file name.